applications/electron/package.json

```diff
@@ -1,19 +1,19 @@
 {
   "private": true,
-  "name": "theia-blueprint",
-  "description": "Eclipse Theia blueprint product",
-  "productName": "Theia Blueprint",
+  "name": "cdt-cloud-blueprint",
+  "description": "CDT.cloud Blueprint is a template for building custom web-based C/C++ tools",
+  "productName": "CDT.cloud Blueprint",
   "version": "1.22.1",
   "main": "scripts/theia-electron-main.js",
   "license": "EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0",
-  "author": "Eclipse Theia <theia-dev@eclipse.org>",
-  "homepage": "https://github.com/eclipse-theia/theia-blueprint#readme",
+  "author": "CDT.cloud <sdirix@eclipsesource.com>",
```

Comment:
@planger Is there some generic mail for cdt cloud which somebody checks we can use? Or shall we keep it my mail for now?

Reply:
Do we have to provide an e-mail address in `author`? In GLSP we use just `author.name`. We use the dev mailing list of the Eclipse project in `contributors` though.

```json
"author": {
  "name": "Eclipse GLSP"
},
"contributors": [
  {
    "name": "Eclipse GLSP Project",
    "email": "glsp-dev@eclipse.org",
    "url": "https://projects.eclipse.org/projects/ecd.glsp"
  }
]
```

Reply:
done